src/test/init.js

```diff
@@ -26,7 +62,7 @@ test('init()', nest => {
     const mutators = ['a', 'b', 'c'].map(
       (i) => (o, { instance }) => { instance[i] = i; });
 
-    const actual = init(...mutators)();
+    const actual = init(mutators[0], mutators.slice(1, 3))();
```

Comment:
Why is the `init` even here? This seems to be clashing with `stampit.init` (which doesn't even work that way). I think there should be clean separation of what are stamp-utils for, not to have a different versions of a same thing.